### ISSUE TITLE
working directory changing during launch

### ISSUE THREAD

**Issue by frankjar** [FredrikNoren/ungit#926]
hi,
 i would use ungit under Windows 8.1 and i have installed it globally with npm globally. But when i launch it it changed the working directory pointing to a "cygwin" one. Here is the console :

> F:\dev\cabestan\sources
> ungit
> Setting log level to warn
> Browse to http://localhost:8448/#/repository?path=F%3A%5Cdev%5Ccabestan%5Csources Took 3095ms to start server.  Ungit started
> 2017-06-20T10:03:09.466Z - warn: Responding with ERROR:  {"isGitError":true,"errorCode":"not-a-repository","command":"-c color.ui=false -c core.quotepath=false -c core.pager=cat remote","workingDirectory":"\\cygdrive\\f\\dev\\cabestan\\sources","error":"fatal: Not a git repository (or any parent up to mount point /cygdrive/f)\nStopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).\n","message":"fatal: Not a git repository (or any parent up to mount point /cygdrive/f)","stderr":"fatal: Not a git repository (or any parent up to mount point /cygdrive/f)\nStopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).\n","stdout":""}

My git repo is `F:\dev\cabestan\sources` and ungit interprets this like `\\cygdrive\\f\\dev\\cabestan\\sources`.

When i change the working directory directly into the browser, ungit transforms it again.

Thx for help

npm v 3.10.10
node 6.9.4
git 2.8.3

**Comment by campersau** [comment]
Are you using cywing to run ungit?
Have you tried to start ungit with cmd or powershell?

**Comment by frankjar** [comment]
Yes i have tried to run ungit under cmd or powershell, with chrome or FF. Same results.
i'm not using cygwin to run ungit, i may think that there is a conflict somewhere...

**Comment by codingtwinky** [comment]
Could you possibly try with unix path separator "/" instead of "\" in the git path entry?

**Comment by rnnr** [comment]
Trying ungit first time and seeing the same error. I'm running it  on Linux and from the project directory (containing  .git dir) but ungit reports the last dir or dirs cut off.
